src/lib/testing/hub-connection.util.ts

```diff
@@ -9,14 +9,15 @@ export interface HeroHub {
 	UpdateHero: string;
 }
 
-export function createSUT(): HubConnection<HeroHub> {
+export function createSUT(maximumAttempts: number, autoReconnectRecoverInterval: number): HubConnection<HeroHub> {
```

Comment:
make as an object instead (not as a deep object)
Can specify the values you have set as the defaults (since i dont see any other usages)

Reply:
applied, and exposed defaults